protocol/x/delaymsg/keeper/dispatch_test.go

```diff
@@ -395,16 +395,23 @@ func TestSendDelayedCompleteBridgeMessage(t *testing.T) {
 
 	aliceAccountAddress := sdk.MustAccAddressFromBech32(constants.BridgeEvent_Id0_Height0.Address)
 
-	// Sanity check: at block 1, balances are as expected before the message is sent.
+	// Sanity check: at block 1, expect bridge balance is expected before the message is sent.
```

Comment:
Nit:

```suggestion
	// Sanity check: at block 1, expect bridge balance is genesis value before the message is sent.
```

or some such.